one_fm/grd/utils.py

```diff
@@ -12,12 +12,6 @@
 from dateutil.relativedelta import relativedelta
 from one_fm.api.notification import create_notification_log
 
-def todo_after_insert(doc, method):
-    doctypes = ['Work Permit', 'Medical Insurance']
-    if doc.reference_type in doctypes and doc.reference_name and doc.owner and 'GRD Operator' in frappe.get_roles(doc.owner):
-        if not frappe.db.get_value(doc.reference_type, doc.reference_name, 'grd_operator'):
-            frappe.db.set_value(doc.reference_type, doc.reference_name, 'grd_operator', doc.owner)
-
```

Comment:
Needs to be preserved this

Reply:
@pjamsheer no need to be stored in todo list as there will be a reminder for the records through their emails